BrowserKit/Sources/WebEngine/WKWebview/Scripts/AdsTelemetryContentScript.swift

```diff
@@ -0,0 +1,114 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Foundation
+import WebKit
+
+public struct EngineSearchProviderModel {
```

Comment:
nit; should we put this in it's own file instead of under the AdsTelemetryContentScript.swift?